arangod/ClusterEngine/ClusterEngine.cpp

```diff
@@ -348,31 +348,14 @@ void ClusterEngine::unloadCollection(
   collection.setStatus(TRI_VOC_COL_STATUS_UNLOADED);
 }
 
-void ClusterEngine::createView(
+Result ClusterEngine::createView(
     TRI_vocbase_t& vocbase,
     TRI_voc_cid_t id,
     arangodb::LogicalView const& /*view*/
 ) {
   THROW_ARANGO_EXCEPTION(TRI_ERROR_NOT_IMPLEMENTED);
```

Comment:
should this now be a return instead of an exception?

Reply:
I think so. At least it would be consistent with what changeView does.